bands/site_sections/band_admin.py

```diff
@@ -24,6 +24,21 @@ def mark_as_band(self, session, group_id):
             'message': '{} has been marked as a band'.format(group.name)
         }
 
+    @ajax
+    def remove_as_band(self, session, group_id):
+        group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
For what it's worth, this if-check isn't really necessary.  We do require that bands have leaders when we assign them, but if we're just un-setting a group as a band, there's probably no reason to do this check.